.drone.yml

```diff
@@ -117,11 +119,12 @@ steps:
     path: /var/run
   - name: tmp
     path: /tmp
-  depends_on:
-  - basic-integration
   when:
-    cron:
-      - nightly
+    event:
+      exclude:
+      - pull_request
```

Comment:
Double check me here. Is this all we need to say "run against any merge to master and on tags"?

Reply:
I'd probably go explicit, 
```
when:
  event:
    - push
    - tag
  branch:
    - master
```

Reply:
updated.